### DISCUSSION TITLE
Russian translation

### DISCUSSION THREAD

**Discussion by Knebergish** [borgbase/vorta#1058] [Translations]
Hello!
I am from Russia, and I would like to help improve the Russian translation.

But when I try to join Vorta on Transifex, there is no Russian in the available languages:
![NoRussian](https://user-images.githubusercontent.com/10116459/131223446-f53dd0f4-b84e-439b-a774-2798786b991b.png)
If I request a new language, Russian is "already added":
![YesRussian](https://user-images.githubusercontent.com/10116459/131223454-b3871190-eec0-4f7f-b3f0-c676bb4859a2.png)

Can someone tell me, am I doing something wrong, or is this some kind of misconfiguration in Transifex?

**Comment by m3nu** [comment]
Hi @Knebergish,

thanks for the interest. I've added you as team member on Transifex. Russian was already there. Let me know if it works as expected now.

**Comment by Knebergish** [comment]
Yes, it works, thank you ).

**Comment by Knebergish** [comment]
@m3nu, I have some questions about the context of using some of the strings. Can I ask them here, or should I use email or another communication channel?
A better understanding of the context would help me translate these strings more accurately.

**Reply by m3nu** [reply]
Here is good, so everyone can read them later.

Usually Transifex will display the source code location of the string, so you can see where it's used.

You can also run the translations locally, as mentioned here: https://vorta.borgbase.com/contributing/translations/

**Reply by Knebergish** [reply]
Yes, having the source code has helped in many cases. But it won't help with these questions:

1. Strings "Form" (№132 in Transifex) and "MainWindow" (234) look like service strings that never appear in the UI. At least, I could not get them to show up. Should these strings be translated and even be in Transifex?
2. Some strings use the word "password" (12, 126, 346...) and some use "passphrase" (23, 225, 335...), although the meaning is the same in my opinion. Is it possible to mean "password" in all these cases? If we are talking about the same thing, we shouldn't use different definitions for it, it would confuse the user.
3. Same as question 2, but for strings "system keychain" (346) and "system password manager" (336) / "password manager" (12, 335...). Should we only use the "system password manager" or "password manager"?
4. The use of capital letters for some words:
Borg (6, 299, 305...) / borg (116, 117)
Profile (93, 121, 259...) / profile (1, 94, 119, 257...)
Repository (6, 14, 122...) / repository (8, 17, 277...)
Archive (142, 176, 221) / archive (50, 70, 104...)
Clipboard (103, 266) / clipboard (161, 211, 267...)
Does that make any sense? Or should we choose one option and use it everywhere?
5. Dots at the end of sentences. Their presence is often made at random ). In many cases I have followed the source, but in some I have done things differently (Misc tab). There must be some centralized strategy.

Part of these issues overlap with #1050 , so they may have already been decided.
I would be glad to hear your opinion on these questions.

**Reply by m3nu** [reply]
I'd prefer to stick to the current strings for now. Passphrase and password can mean different things in different context. It needs a very close look.

Agree with all the issues you found. I didn't prioritize these kinds of things most of the time. But feel free to make small PRs to improve the situation.
